README.md

```diff
@@ -1,33 +1,45 @@
-# Flight HUD
+<p align=center>
+    <img src="images/logo.png">
+</p>
 
-__Flight HUD__ is a client side Fabric Minecraft Mod that adds a flight style HUD like one you would see in a flight
-simulator.
+<p align=center>
+    <a href="https://modrinth.com/mod/flightassistant">
+        <img src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3.1.2/assets/cozy/available/modrinth_vector.svg"
+            alt="Available on Modrinth"></img></a>
+    <a href="https://modrinth.com/mod/fabric-api/">
+        <img src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/requires/fabric-api_vector.svg"
+            alt="Requires Fabric API"></img></a>
+    <a href=""> <!-- don't open image preview -->
+        <img src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3.1.2/assets/cozy/unsupported/forge_vector.svg"
+            alt="Won't support Forge"></img></a>
```

Comment:
hold up

Reply:
(Neo)Forge support is possible in the future, asserting that there won't be a Forge version is too much